docs/intentReaderHandlers/SetApiResponseHandler.md

```diff
@@ -0,0 +1,92 @@
+# SetApiResponseHandler 
+
+## Overview
+
+SetApiResponseHandler is invoked when the task specified in the intent has the value "setApiResponse". This handler is used to assign the apiResponse value to an environment variable in FCA.
+
+It performs the following actions :
```

Comment:
Can we remove the space between actions & :

It performs the following actions:

Reply:
updated for all handlers